bids/analysis/analysis.py

```diff
@@ -12,15 +12,15 @@
 
 
 class Analysis(object):
-    ''' Represents an entire BIDS-Model analysis.
+    """Represents an entire BIDS-Model analysis.
 
     Args:
```

Comment:
Should these become `Parameters`?

Reply:
🤦‍♂ I totally missed analysis.py and auto_model.py. Thanks for catching that.